test-suite/Makefile

```diff
@@ -102,7 +102,7 @@ VSUBSYSTEMS := prerequisite success failure $(BUGS) output \
   coqdoc ssr
 
 # All subsystems
-SUBSYSTEMS := $(VSUBSYSTEMS) misc bugs ide vio coqchk coqwc coq-makefile tools unit-tests
+SUBSYSTEMS := $(VSUBSYSTEMS) misc bugs ide vio coqchk coqwc coq-makefile unit-tests #tools
```

Comment:
Not sure what this means...

Reply:
See the comment on the last commit. Working with this or I have discovered some tests that are either broken or too fragile. The current one in tools certainly seems very suspicious, in particular due to the way it access the documentation, and in how many path deps it introduces, I guess I'll file a bug.

Reply:
Yep, I had read the commit message but it was still not clear to me. Anyways, this change should be reviewed by @gares.

Reply:
In fact I finally understood thanks to your comment in another thread:

>For example, the update-compat test is quite fragile and it even depends on the stdlib documentation having been built.

Reply:
Yeah, sorry if things are not very clear. As this depends on a few PR I hesitated to improve more as to avoid spam.